stream_alert/apps/_apps/README.rst

```diff
@@ -35,8 +35,8 @@ How to update necessary dependencies
 ####################################
 
 
-Create an EC2 Instance
-======================
+Build dependencies on an EC2 Instance (python3.7)
```

Comment:
I don't think we need the python3.7 reference.. just say:

`Building Dependencies Using EC2`